### ISSUE TITLE
wardriving not loading

### ISSUE THREAD

**Issue by MikePerez** [offensive-security/kali-nethunter#716]
### Device:
Nexus 7 2013 deb

### OS version (KitKat/Lollipop/Marshmallow/Nougat):
Marshmallow - 6.0.1

### Built from repo (date and build command) or downloaded from website (links):
https://images.offensive-security.com/nethunter-release/nethunter-flo-marshmallow-3.0.zip

## Issue:
Wardriving not starting up. ran kismet from kali terminal in nethunter and got /usr/bin/kistmet_client:  symbol lookup error: undefined symbol Y.

**Comment by vay3t** [comment]
please, try build nethunter image with build.py

**Comment by MikePerez** [comment]
Git seems to be busy, I can't clone the repo, but I'll try again in a few.  Also, I don't see a version for nexus 7 LTE, but on the main site, it says its supported? Unless someone forgot to update  the site? https://www.offensive-security.com/kali-linux-nethunter-download/

**Comment by akrolla** [comment]
they may have dropped support for the LTE version. They recently stopped
supporting the wifi variant.

**Comment by MikePerez** [comment]
I think you are right... oh man! that sucks, I guess I'll have to flash again and go back to Pwnpad. 
thanks Akrolla!

**Comment by akrolla** [comment]
I had to do the same thing. if u can find a 3.4 kernel that has packet
injection it is possible to build one but I have yet to find a 3.4 kernel
with packet injection. A couple of us have searched high and low for one
before they dropped it..

**Comment by akrolla** [comment]
Check this out

This was when I was searching
https://github.com/offensive-security/kali-nethunter/issues/639

**Comment by akrolla** [comment]
That should give u some insight on how to go about it, it is possible. I
just dont have time , goimg to school and family. But if u so find a
workable kernel and are able to get wverything functional for the love of
god let me know. I like aopp the GUI is nice but nethunter is wayyy better.

Plus aopp is based partially @binkybears old work.. before the creation of
this project... nethunter has waynore functionality .

**Comment by MikePerez** [comment]
I hear ya! I'll give it a shot after the holidays. I'm in the same boat, family and school takes 90% of my energy, the last 10% its for small stuff.

**Comment by akrolla** [comment]
Yep. The struggle is real. That link I posted was to my search. And
@Jcadduono pretty much gives you a way to do it. By the time he posted that
I had already installwd aopp , so I never tried, check out the KALI PWN
PAD. I didn't go that route either but I hear its good, and its supposedly
the predecessor to this project but I dunno

encrypted.google.com/url?sa=t&source=web&cd=1&ved=0ahUKEwjVg5XI8YPRAhWLrlQKHX61A3gQFggZMAA&url=http%3A%2F%2Fforum.xda-developers.com%2Fshowthread.php%3Ft%3D2577356&usg=AFQjCNFLLSQ5V_X5FrUHHaqAv6yEL4rNqQ

**Comment by jcadduono** [comment]
is https://build.nethunter.com/nightly/3.15.3-20161220-0940/kernel-nethunter-flo-marshmallow-3.15.3-20161220-0940.zip not working for you?

**Comment by MikePerez** [comment]
Thanks @jcadduono !! I will flash again, probably tonight, so I can get back to you asap. But what I was facing last time was when starting up wardriving it didn't ran and when typed kismet from the terminal it gave me a weird error and didn't detected my TP-Link. 

thanks for the help! 

-Mike Perez

**Comment by akrolla** [comment]
Oh he has flo? Roger that

**Comment by MikePerez** [comment]
@akrolla its 2013 but its LTE which is Deb. flo its the Wifi only version and deb its the Wifi+LTE.

**Comment by MikePerez** [comment]
First, thank you for the quick help!! 

ok, so here are the steps I took last night.

flash marshmallow 6.0.1 again and wiped the pwnpad image (https://developers.google.com/android/images#razorg)  -  boots find

flash recover image TWRP 3.0.2-0

installed thru TWRP Kernel link from above  - boots find

installed thru TWRP nethunter from link above - boots find 

reinstalled thru TWRP Kernel link from above - boots find

At this point I opened every single tool to make sure they opened. Wardriving opened just find - We are all good at this point and beyond the previous issue. 

Now the real test - installed EZ file explorer app to test OTG cable. connected OTG cable with USB stick, connected and open just find. 

Connected TP-Link TL-WN722N opened Nethunter> wardriving> start kismet and nothing... 

It detect the TP-Link and it start kismet, but once there, its not detecting any network around my area and no GPS data. I can see location its on and on the actually wardriving app page, I see GPS coordinates running before I stop the GPS, but Kismet cannot detect anything. 

The funny part its that I don't see the TP-Link blinking at all...

I tested the TPLink on computer and with another OTG cable just in case it was the cable and nothing. It also worked on the same hardware but with the previous (aopp) image... 

Any leads I could chase ?
